### DISCUSSION TITLE
blog/zh/reading/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#10] [Announcements]
# blog/zh/reading/

Ladder@记得笔者刚上大学的时候，尤其讨厌将阅读作为一种方法来讨论研究。认为读书唯求愉快，只在乎共鸣和是否获得了享受。当时笔者更追求的是读书不求甚解，对做研究般的读书方法是尽显鄙夷之态，对咬文嚼字般的读法更是嗤之以鼻。
时过境迁，后续接触到了更复杂的学科，更加晦涩的知识后，对于知识也就有了更多的敬畏，并且慢慢发现自己的阅读方法和态度跟不上知识的要求，于是开始慢慢的探索起来，将目光重新放回到了阅读的方法论中，也就有了这一篇文章分享。

https://guangzhengli.com/blog/zh/reading/

**Comment by tangyunlong** [comment]
“把学到的东西藏在心里不仅是可耻的，而且是有害的，凡是我们不去努力分享的，也必将从我们这里失去。”
受益匪浅。salute

**Comment by wonderfultina** [comment]
notion好漂亮 求模版

**Reply by guangzhengli** [reply]
文章中有链接哈，模版地址：https://guangzhengli.notion.site/Guangzheng-Book-Template-a446554b015748deb4c61559f99f9bf8

**Comment by Junyi-99** [comment]
写得好棒，作者是 PhD 吗？

**Comment by rukal** [comment]
Open Syllabus Project，这个绝对是个好东西啊，感谢分享！

**Comment by weey007** [comment]
哈哈，[如何阅读一本书 (豆瓣)](https://book.douban.com/subject/1013208/) 也是这么说的
